pkg/machines/components/storagePools/storagePoolDelete.jsx

```diff
@@ -86,33 +131,86 @@ export class StoragePoolDelete extends React.Component {
+                    </span>
+                );
+            }
+        };
+
         let defaultBody = (
-            <div className='ct-form'>
-                { storagePool.active && volumes.length > 0 && <React.Fragment>
-                    <label className='control-label'>
-                        {_("Delete Content")}
-                    </label>
-                    <label className='checkbox-inline'>
-                        <input id='storage-pool-delete-volumes'
-                            type='checkbox'
-                            checked={this.state.deleteVolumes}
-                            onChange={e => this.onValueChanged('deleteVolumes', e.target.checked)} />
-                        {_("Delete the Volumes inside this Pool")}
-                    </label>
-                </React.Fragment>}
-                { !storagePool.active && _("Deleting an inactive Storage Pool will only undefine the Pool. Its content will not be deleted.")}
-            </div>
+            <React.Fragment>
+                <div className='ct-form'>
+                    { storagePool.active && volumes.length > 0 && <React.Fragment>
+                        <label className='control-label'>
+                            {_("Delete Content")}
+                        </label>
+                        <div role="group">
+                            <label className='checkbox-inline'>
+                                <input id='storage-pool-delete-volumes'
+                                    type='checkbox'
+                                    checked={this.state.deleteVolumes}
+                                    onChange={e => this.onValueChanged('deleteVolumes', e.target.checked)} />
+                                {_("Delete the Volumes inside this Pool")}
+                            </label>
+                        </div>
+                    </React.Fragment>}
+                    { !storagePool.active && _("Deleting an inactive Storage Pool will only undefine the Pool. Its content will not be deleted.")}
+                </div>
+                { storagePool.active && showWarning() }
+            </React.Fragment>
         );
 
+        const deleteButton = () => {
+            if (!canDelete(storagePool, vms)) {
+                return (
+                    <OverlayTrigger overlay={
+                        <Tooltip id='delete-tooltip'>
+                            {_("Pool's volumes are used by VMs ")}
+                            <b> {vmsUsage + "."} </b>
+                            {_("Detach the disks using this pool from any VMs before attempting deletion.")}
+                        </Tooltip> } placement='top'>
+                        <span className="d-inline-block">
+                            <Button id={`delete-${id}`}
+                                bsStyle='danger'
+                                style={{ pointerEvents: 'none' }}
+                                onClick={this.open}
```

Comment:
You don't really need onClick here since it's disabled. No need to change, just for heads up.